api/olsconfig-api.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olsconfig-api"]
+= OLSConfig API reference
+include::_attributes/common-attributes.adoc[]
+:context: olsconfig-api
```

Comment:
Where is this assembly used?

Reply:
@mburke5678 this was my mistake. I meant to delete this assembly. Originally, I created this assembly as a top-level category. After presenting two ideas to the team, I decided to move the API reference into the existing configuration category. I will clean up the PR. Thanks for noticing this!

Reply:
Deleted the file, pushed the change and rebased.